lib/segment/src/vector_storage/quantized/quantized_multivector_storage.rs

```diff
@@ -0,0 +1,180 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::marker::PhantomData;
+use std::path::Path;
+
+use common::types::{PointOffsetType, ScoreType};
+use quantization::{EncodedVectors, VectorParameters};
+use serde::{Deserialize, Serialize};
+
+use crate::common::operation_error::{OperationError, OperationResult};
+use crate::data_types::vectors::TypedMultiDenseVectorRef;
+use crate::types::{MultiVectorComparator, MultiVectorConfig};
+
+#[derive(Clone, Copy, Debug, Default, Serialize, Deserialize, PartialEq)]
+pub struct MultivectorOffset {
+    pub offset: PointOffsetType,
+    pub count: PointOffsetType,
+}
+
+pub struct QuantizedMultivectorStorage<TEncodedQuery, QuantizedStorage>
```

Comment:
Quantized multivectors container. It aggregates SQ/PQ/BQ for all inner vectors and contains offsets